genesyscloud/data_source_genesyscloud_knowledge_knowledgebase.go

```diff
@@ -0,0 +1,64 @@
+package genesyscloud
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/mypurecloud/platform-client-sdk-go/v80/platformclientv2"
+)
+
+func dataSourceKnowledgeKnowledgebase() *schema.Resource {
+	return &schema.Resource{
+		Description: "Data source for Genesys Cloud Knowledge Base. Select a knowledge base by name.",
+		ReadContext: readWithPooledClient(dataSourceKnowledgeKnowledgebaseRead),
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "Knowledge base name",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"core_language": {
```

Comment:
Usually we just use the name to search. Is there a reason for also using the core_language?

Reply:
I included both because the knowledge base creation resource `POST
/api/v2/knowledge/knowledgebases` lists "name" as optional but "core_language" as required.  Which doesn't really make sense to me, but I was thinking this should guarantee uniqueness.

Reply:
This doesn't guarantee uniqueness, though.
Two solutions I can think of:
1) make "name" required for the resource even though it's optional in the API
2) just not have a data source for this resource

1) makes most sense to me, any ideas?

Reply:
No that's fine. Good to know the reasoning behind it